README.md

```diff
@@ -3,131 +3,219 @@
+
+#### Does Membrane videoroom support broadcasting (only one peer streaming and many peers who are only watching)?
+Membrane videoroom demo is an application meant to mimic the behavior of a videoconferencing room (i.e. Google Meet). It has mechanisms to reduce the amount of resources consumed when only part of the video conference participants are actively participating (streaming the multimedia), and the rest are only watching the stream.
+At the same time, for broadcasting multimedia to thousands of viewers, other mechanisms need to be used, and it can be achieved with the use of the Membrane Framework. We invite you to take a look at other [Membrane demos](https://github.com/membraneframework/membrane_demo), especially:
+* [WebRTC to HLS demo](https://github.com/membraneframework/membrane_demo/tree/master/webrtc_to_hls)
+* [RTMP to HLS demo](https://github.com/membraneframework/membrane_demo/tree/master/rtmp_to_hls)
+which are demo applications meant to broadcast the media sent by the streamer to many viewers.
+<!--> ADD INFORMATION ABOUT RTMP TO HLS TUTORIAL WHEN IT IS PUBLISHED <-->
+
+### Configuration
+#### Can you use the external TURN server instead of the integrated one?
+It's not possible at the moment. Since the SFU needs to be publicly available, we don't see a purpose of having a separate publicly available TURN server.
+The architecture proposed by us: `client --- NETWORK ---> SFU with TURN` reduces the number of 'hops' in the network by one. In contrast, the architecture with the separate TURN server would like that: `client --- NETWORK -- TURN --- NETWORK ---> SFU`.
+If you can think of a use case for a separate TURN server, feel free to start a discussion on our discord server.
+
+### Known bugs
+#### `(Mix) Could not compile dependency :fast_tls,`
+As the error suggests, there was a problem with the `:fast_tls` dependency compilation. It's usually caused by the fact, that the compiler cannot
+find OpenSSL dependencies in your system.
+In the setup instruction above we have set the compiler flags pointing to the appropriate directories containing OpenSSL dependencies on a given OS.
+However, if you have your OpenSSL installed in some custom location, you will need to change the compiler flags (i.e. [`LDFLAGS`, `CFLAGS`, `CPPFLAGS`] or [`C_INCLUDE_PATH`, `LIBRARY_PATH`], depending on your compiler and OS) and the `PKG_CONFIG_PATH` environment variable.
+For more instructions on how to install the `:fast_tls` dependency, please visit the [Fast TLS repository](https://github.com/processone/fast_tls).
+
+#### Why, after joining the room, I don't see a video stream from the others?
+That might be due to the misconfiguration of the `EXTERNAL_IP` environment variable. Make sure, that the variable is set in the environment where you are running the server, as well as that it is the IPv4 address pointing to the server, visible by all the peers.
+
+#### When I run the videoroom in the docker container, why cannot I access VP8 tracking?
```

Comment:
What is VP8 tracking? I don't understand the question at this point.

Reply:
Oh, that was the term used in the question asked on Discord. At first I thought that somebody had problems with displaying metrics at `chrome://webrtc-internals`, but apparently the problem was with a video not displaying at all, so I've decided to remove that question from FAQ.